### PR TITLE
Close modal when submit is clicked

### LINKED ISSUE #511: Advanced form warning does not close on submit
When submitting in advanced mode with use public key checked the final confirmation modal does not close. It should close when the submit button is clicked.

https://github.com/ScopeLift/umbra-protocol/pull/494/files#diff-428f8140870a026c3dd9dc90b835bd0a7deb07a7c8bb8f629ee8e7f9f7156baaR624
image

_Originally posted by @mds1 in https://github.com/ScopeLift/umbra-protocol/pull/494#discussion_r1197941494_

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

When submitting in advanced mode with use public key checked the final confirmation modal does not close. Now, it should close.

Closes #511

### CODE REVIEW

File: frontend/src/pages/AccountSend.vue
Comment (mds1) [inline frontend/src/pages/AccountSend.vue:1]: 1. It looks like we show the "warn about checkbox" modal twice: Once when the "Send using recipient's standard public key" box is checked, and again when hitting send. I confirmed this behavior was not changed on this PR. Regardless, I can't remember if this was intentional/desired? @apbendi do you remember?

2. Additionally, after the "advanced with box checked" send completes, it seems the "send" button stays disabled. I don't recall this happening on main but I did not verify

3. As shown below, the "send" button does not enter the "disabled with loading spinner" state immediately after I click it, but it should. In the below image you can see button is not disabled even though MM is open. Once I sign, then it enters the "disabled with loading spinner" state. I don't think this was introduced on this PR, but just mentioning it and @alexkeating will let you decide if you want to fix it here or in a separate PR

image

Comment (alexkeating) [reply frontend/src/pages/AccountSend.vue:1]: 1. From my memory this is intentional, #362 
2. I may be confused here. But, after the send completes the form is cleared which I think is what is causing the send to be disabled. From my understanding this is expected behavior in the app.
3. I can fix this in this pr. It should be as simple as moving 1 line

Comment (mds1) [reply frontend/src/pages/AccountSend.vue:1]: 1. Got it
2. Ah right of course 🤦
3. Thakns!

**Review by garyghayrat — APPROVED** [review]
🚢 it

File: frontend/src/pages/AccountSend.vue
Comment (alexkeating) [reply frontend/src/pages/AccountSend.vue:1]: https://github.com/ScopeLift/umbra-protocol/pull/595/commits/249ec718d12919c7d1801e76acf3161f750eb99b

**Pull request merged**